README.md

```diff
@@ -141,6 +141,41 @@ That includes warnings found by rustc (e.g. `dead_code`, etc.). If you want to a
 an error for Clippy warnings, use `#![deny(clippy::all)]` in your code or `-D clippy::all` on the command
 line. (You can swap `clippy::all` with the specific lint category you are targeting.)
 
+### Install from source
```

Comment:
Should we put this in the readme? This is more for developers rather than users. I'd put this somewhere in `doc/`.

Reply:
Yeah, I can tack it on the end of doc/basics.md